infrastructure/shareable-lists-api/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "extends": "tsconfig/cdktf.json",
+  "compilerOptions": {
+    "outDir": "dist",
+    "rootDir": "src"
+  },
+  "exclude": ["node_modules"],
+  "include": [
+    "src/**/*.ts",
+    "src/config",
+    "src/files/nodejs/node_modules/jmespath/*.js",
+    "src/files/nodejs/node_modules/synthetic/*.ts"
```

Comment:
interesting

Reply:
Yes, I wasn't sure what this was. I assumed related to the synthetic check, and made a note to look into it after.